[devops] Add deployment files and info (#1)
antomor/rif-relay-server: deploy/README.md

> # Deployment information

Reviewer: Thank you for this description, if you don't mind, I'd add also some instructions in the main README.

* Information for deploying the application to a server,
not meant to be used locally.
* Server must have all the files referenced here and 

> should have and rsk node installed and running locally. 

Reviewer: ```suggestion
should have an RSKj node installed and running locally. 
```

## Build and deploy
```
sudo docker-compose build --no-cache
sudo docker-compose up -d --force-recreate
```

## In this directory

> * ``Dockerfile``: The Dockerfile for the application on server.

Reviewer: ```suggestion
* `Dockerfile`: The Dockerfile for the server application.
```

> * ``docker-compose.yml``: All Docker services configuration on server.

Reviewer: ```suggestion
* `docker-compose.yml`: All Docker services configuration.
```

> * ``.env``: Environment variables for the application.

Reviewer: I think it's intended to be used for the dapp

Author: Erased, thanks
